pmlc/target/x86/stencil_tpp_unary.cc

```diff
@@ -129,12 +143,60 @@ class StencilImpl : public pxa::StencilBase {
     this->opName = inName;
   }
 
+  void maybeCaptureReduceOp(Optional<pxa::StencilCapture> &capture,
+                            StringRef inName, arith::AtomicRMWKind agg) {
+    if (capture)
+      return;
+    AffineValueMap ranges = util::getRangesValueMap(op);
+    auto constExpr =
+        ranges.getResult(op.getIVs().size() - 2).dyn_cast<AffineConstantExpr>();
+    if (constExpr && constExpr.getValue() > 1) {
```

Comment:
Be consistent with braces (see line 148). For single stmt I prefer not to have them.